### PR TITLE
[ASM] [DOCS-7629] new ASM Attacker Explorer topic

### PR DESCRIPTION
**Pull request by jkirsteins**
### What does this PR do? What is the motivation?

2 files changed:

- Threats Overview changed to Attack Summary
- New, brief Attachers Explorer topic

Jira: https://datadoghq.atlassian.net/browse/DOCS-7629

### Merge instructions

- Do not merge after reviews. Still waiting for feature release.

### Additional notes

UI of feature changes: 
https://dd-7ea7601508ea5d29f7a595e19234be38.datad0g.com/security/appsec/attackers

### CODE REVIEW

File: content/en/security/application_security/threats/attackers.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# Understanding Attackers
+
+ASM classifies IP addresses as
+Benign
+Suspicious
+Flagged
+
+ASM additionally clusters IPs with similar attributes together.
+
+Benign: Traffic from APM that has not recently sent attack traffic
```
Comment (emmlejeail) [inline content/en/security/application_security/threats/attackers.md:15]: Should we reformulate like this to be clearer?
```
Benign: IPs that haven't sent any significant amont of attack attempts
```

File: content/en/security/application_security/threats/attackers.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+{{< img src="security/application_security/threats/attackers/attacker_explorer_default_view.png" alt="Screenshot of the ASM Attacker Explorer"  >}}
+
+# Clusters
+{insert blurb on clusters}
+
+# Relationship to Signals and Traces
+Evidence of an attack is visible on the Trace Explorer, which identifies specific payloads that are threats.
+
+Signals correlate traces into actionable alerts.
```
Comment (emmlejeail) [inline content/en/security/application_security/threats/attackers.md:31]: Maybe like this?
```suggestion
Signals aggregate attack attempts into actionable alerts.
```

File: content/en/security/application_security/threats/attackers.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Attack Explorer identifies attackers with persistent threats which may cross multiple Signals.
```
Comment (emmlejeail) [inline content/en/security/application_security/threats/attackers.md:33]: And then this?
```suggestion
The Attacker Explorer detects persistent threats from attackers that could span across multiple Signals.
```

Comment (jkirsteins) [reply content/en/security/application_security/threats/attackers.md]: Updated based on latest draft, which removes this.

File: content/en/security/application_security/threats/attackers.md (hunk 2 above)
Comment (jkirsteins) [reply content/en/security/application_security/threats/attackers.md]: Updated based on latest draft, which removes this.

File: content/en/security/application_security/threats/attackers.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+Suspicious and Flagged are available as facets. Benign traffic is shown when evaluating the impact of blocking an IP.
+
+{{< img src="security/application_security/threats/attackers/attacker_explorer_default_view.png" alt="Screenshot of the ASM Attacker Explorer"  >}}
+
+# Clusters
+{insert blurb on clusters}
+
+# Relationship to Signals and Traces
+Evidence of an attack is visible on the Trace Explorer, which identifies specific payloads that are threats.
```
[the remark this answers is not in the document]
Comment (jkirsteins) [reply content/en/security/application_security/threats/attackers.md]: Updated based on latest draft, which removes this.

File: content/en/security/application_security/threats/attackers.md (hunk 1 above)
Comment (jkirsteins) [reply content/en/security/application_security/threats/attackers.md]: Updated based on latest draft, which removes this.

**Comment by michaelcretzman** [comment]
I've revised this PR but I'll need to revisit it once the UI is finalized. 

There are a number of TODOs in it.

**Review by michaelcretzman — APPROVED** [review]

**Pull request merged**